GPS[] Path
uint16 PathID
uint64 DroneID
bool loiterAtEnd